src/Inciter/Carrier.C

```diff
@@ -908,6 +927,11 @@ Carrier::aec( const tk::Fields& Un )
       for (auto i : n.second) p.push_back( m_p[ tk::cref_find(m_lid,i) ] );
       thisProxy[ n.first ].comaec( n.second, p );
     }
+
+  ownaec_complete();
+  #ifndef NDEBUG
+  ownaec_complete();
```

Comment:
While having special flags for debugging is always nice, having two separate control paths and communication requirements between the debug and mainline path is a recipe for disaster and a way to find very hard to debug problems as we are fundamentally changing the logic of the program when we debug.